lumigator/python/mzai/backend/Dockerfile.uv

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11-slim-bookworm
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
+
+# Copy the project into the image
+ADD . /app
+
+WORKDIR /app
+
+# Sync the project into a new environment, using the frozen lockfile
+RUN uv sync --frozen
+
+CMD ["uv", "run", "uvicorn",  "--host", "0.0.0.0", "--port", "8000", "backend.main:app"]
```

Comment:
Maybe make the port configurable vía env var?

Reply:
Ideally, at runtime. But this means using a script as CMD and moving the current starting point there, we can leave that for later.